csvcubed/csvcubed/schema/codelist-config/v1_0/codelistconfig-example.jsonc

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "https://purl.org/csv-cubed/code-list-config/v1.0",
```

Comment:
Can we also support `https://purl.org/csv-cubed/code-list-config/v1` as well which should default to the latest minor version of the v1.* code-list config.

I have created the above URL redirecting to the v1.0 version at the moment.